lms/djangoapps/instructor/tests/test_spoc_gradebook.py

```diff
@@ -169,3 +170,36 @@ def test_assigned_grades(self):
         # User 0 has 0 on the class [1]
         # One use at the top of the page [1]
         self.assertEquals(3, self.response.content.count('grade_None'))
+
+
+@attr('shard_1')
+class TestPassingGrade(TestGradebook):
+    """
+    Tests advanced grading policy (with letter grade cutoffs and passing grades).
+    """
+    grading_policy = {
+        "GRADER": [
+            {
+                "type": "Homework",
+                "min_count": 1,
+                "drop_count": 0,
+                "short_label": "HW",
+                "passing_grade": .9,
+                "weight": 1
+            },
+        ],
+        "GRADE_CUTOFFS": {
+            'A': .9,
+            'B': .8,
+            'C': .7,
+            'D': .6,
+        }
+    }
+
+    def test_assigned_grades(self):
+        # Users 9-10 have >= 90% on Homeworks [2]
+        # Users 9-10 have >= 90% on the class [2]
+        # One use at the top of the page [1]
```

Comment:
> Are those numbers correct?

What numbers? From comment above?
BTW this test we left as is to fix it during code review.
This test is not passed.
`self.assertEquals(19, self.response.content.count('grade_None'))` wrong - there is only 3 times 'grade_None' encounters.
Code for `lms/templates/courseware/gradebook.html` was added but test still not passes.